donut/modules/uploads/templates/uploaded_list.html

```diff
@@ -0,0 +1,30 @@
+{% extends "layout.html" %}
+{% block scripts %}
+  {{super()}}
+  <script src="https://cdn.rawgit.com/showdownjs/showdown/1.0.1/dist/showdown.min.js"></script>
+{% endblock %}
+{% block page %}
+<div class="container">
+  <div class="col-md-24">
+    <div class="jumbotron">
+      <h1 id = "title" class="text-center">List of Uploaded Pages</h1>
+      <div id = "content">
+        <table class = "table">
+          <tr>
+            <th>File name</th>
+            <th> </th>
+          </tr>
+          {% for link, name in links %}
+            <tr>
+              <td><a href= "{{ link }}">{{name}}</a></td>
+              {% if permissions%}
+                <td><a href= "{{ url_for('uploads.delete_uploaded_file', filename=name)}}">Delete</td>
+              {% endif %}
+            </tr>
+          {% endfor %}
+        </table>
+      </div>
+    </div>
+  </div>
+</div>
+{% endblock %}
```

Comment:
This template looks very similar to the one for `editor`. Is it possible to merge them into one template?

Reply:
hmmm by some sketchy ways (modifying `current_app.jinja_loader`) I can load this template in `editor`. However, it doesn't seem to be able to find `layout.html` from which it inherits from. 

So I'm going to move one of the routes to the other folder for now? There might be better ways that I can't seem to find.

Reply:
Hmm, I think you might be able to symlink one of the templates to the other so there is only one copy

Reply:
Yeee it worked